src/app/services/info/info.service.ts

```diff
@@ -58,4 +61,19 @@ export class InfoService {
       );
   }
 
+  private getSegregatedData = (category: string, id: string): Observable<object> => {
+    return this.http.get(`/server/info/people/alumni?${category}=${id}`)
```

Comment:
and try make the function generic for any info, not just alumni.
```suggestion
    return this.http.get(`/server/info/people/${category}/${id}?${field}=${value}`)
```